sources/contrib/com.polyphonesoundfonts.polyphone.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Polyphone Developers -->
+<component type="desktop">
+ <id>com.polyphonesoundfonts.polyphone</id>
```

Comment:
Should be `com.polyphone_soundfonts.polyphone` according to [the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic), here and everywhere else.

Reply:
Mmm. You are right.

I'll update the PR with this addressed. (and the flatpak while I'm at it)

Thanks!

Reply:
Addressed in this PR